src/net_processing.cpp

```diff
@@ -4505,7 +4506,7 @@ bool PeerLogicValidation::SendMessages(CNode* pto)
         if (state.m_tx_download.m_check_expiry_timer <= current_time) {
             for (auto it=state.m_tx_download.m_tx_in_flight.begin(); it != state.m_tx_download.m_tx_in_flight.end();) {
                 if (it->second <= current_time - TX_EXPIRY_INTERVAL) {
-                    LogPrint(BCLog::NET, "timeout of inflight tx %s from peer=%d\n", it->first.ToString(), pto->GetId());
+                    LogPrint(BCLog::NET, "timeout of inflight tx %s from peer=%d\n", it->first.GetHash().ToString(), pto->GetId());
```

Comment:
Could have a `GenTxid::ToString` which specifies whether the hash is a wtxid or a regular txid

Reply:
Done.